content/docs/04-clusters/03-edge/09-site-deployment/05-site-installation/07-cluster-deployment.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Create Cluster Definition"
+metaTitle: "Create Cluster Definition"
+metaDescription: "Define your Edge cluster using the Edge hosts that are registered and available."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview
+
+To complete the Edge Installation process, an Edge host must become a member of a host cluster. You can add an Edge host to an existing host cluster that is of the type Edge Native, or you can create a new host cluster for Edge hosts to become a member.
+
+
+
+# Create an Edge Native Host Cluster
+
+Use the following steps to create a new host cluster so that you can add Edge hosts to the node pools.
+
+## Prerequisites
+
+- A registered Edge host.
+
+## Create Cluster
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Navigate to the left **Main Menu** and select **Clusters**.
+
+
+3. Click on **Add New Cluster**.
+
+
+4. Choose **Edge Native** for the cluster type and click on **Start Edge Native Configuration**.
+
+
+5. Provide the cluster with a name, description and tags. Click on **Next**.
+
+
+6. Select a cluster profile. If you don't have a cluster profile for Edge Native, refer to the [Create Edge Native Cluster Profile](/clusters/edge/site-deployment/model-profile#createedgenativeclusterprofile) guide. Click on **Next** after you have selected a cluster profile.
+
+
+7. Review your cluster profile values and make any changes as needed. Proceed to the next step by selecting **Next**.
+
+
+8.  Provide the host cluster with the Virtual IP address used by the physical site. You can also select any SSH keys in case you need to remote into the host cluster. And you can provide a list of Network Time Protocol (NTP) servers. Click on **Next**.
```

Comment:
```suggestion
8.  Provide the host cluster with the Virtual IP (VIP) address used by the physical site. You can also select any SSH keys in case you need to remote into the host cluster. And you can provide a list of Network Time Protocol (NTP) servers. Click on **Next**.
```